### PR TITLE
Add UI turn messages 

### PR DESCRIPTION
**Pull request by sabljak**

### CODE REVIEW

File: Assets/Scripts/UI/Menus.cs [hunk 1]
```diff
@@ -11,12 +13,27 @@ public class Menus : MonoBehaviour {
...
+    public GameObject turnMessagesUI;
+    public GameObject turnMessageText;
     public bool isGamePaused = false;
     public NetworkManager networkManager;
     public Camera UICamera;
 
     private void Start() {
         toggleInputScript(false);
+        MovementLogic.FigureMoved += changeTurnMessage;
+    }
```
Comment (nedzadalibegovic) [inline Assets/Scripts/UI/Menus.cs:25]: Add line after brace

Comment (sabljak) [reply Assets/Scripts/UI/Menus.cs:25]: Say magic words

Comment (nedzadalibegovic) [reply Assets/Scripts/UI/Menus.cs:25]: Pretty please 🥺

Comment (sabljak) [reply Assets/Scripts/UI/Menus.cs:25]: That's more like it

**Pull request merged**